extensions/sns/build.rs

```diff
@@ -0,0 +1,34 @@
+use std::path::PathBuf;
+
+const REPLICA_REV: &str = "90e2799c255733409d0e61682685afcc2431c928";
+
+const BINARY_DEPENDENCIES: &[(&str, &str)] = &[
+    // (downloaded binary name, renamed binary name)
+    ("sns", "sns-cli"),
+];
+
+fn main() {
+    // keep copy of the dependency in the root of the project, so that cargo-dist will be able to package it into a tarball
+    let manifest_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    // and also in `target/debug` or `target/release` for development purposes (e.g. cargo run)
+    let target_dir = manifest_dir
```

Comment:
I'm no expert, but is this `CARGO_TARGET_DIR`?

Reply:
it is `CARGO_TARGET_DIR`, but `CARGO_TARGET_DIR` is not available when running build script